src/parser/rogue/shared/resources/EnergyDetails.js

```diff
@@ -4,24 +4,64 @@ import Analyzer from 'parser/core/Analyzer';
 import Tab from 'interface/others/Tab';
 import StatisticBox, { STATISTIC_ORDER } from 'interface/others/StatisticBox';
 import Icon from 'common/Icon';
+import { formatPercentage } from 'common/format';
 import ResourceBreakdown from 'parser/shared/modules/resourcetracker/ResourceBreakdown';
 
 import EnergyTracker from './EnergyTracker';
 
+const MINOR_THRESHOLD = 0.05;
+const AVERAGE_THRESHOLD = 0.1;
+const MAJOR_THRESHOLD = 0.2;
+
 class EnergyDetails extends Analyzer {
   static dependencies = {
     energyTracker: EnergyTracker,
   };
 
+  get wasted() {
+    return this.energyTracker.wasted || 0;
+  }
+
+  get total() {
+    return this.energyTracker.wasted + this.energyTracker.generated || 0;
+  }
+
+  get wastedPercent() {
+    return this.wasted / this.total || 0;
+  }
+
+  get suggestionThresholdsWasted() {
```

Comment:
Is currently unused, a copy&paste error or are you planning on adding a suggestion for this?